src/MainPage.js

```diff
@@ -0,0 +1,32 @@
+import logo from './images/logo.jpg';
+import './Styles.css';
+import { Link } from 'react-router-dom';
+
+function MainPage() {
+    return (
+        <>
+            <div className="container">
+                <div className="centered-content">
+                    <image className="logoImage" src="/src/images/logo.jpg" alt="Logo"></image>
+                    <h1> Elder Friendly Chat</h1>
+
+                    <div className="centered-buttons">
+                        <img className="img" src="/src/images/conversations1.png" alt="conversations"></img>
+                        <Link to="/conversations"><button class="button"> View conversations</button></Link>
+                    </div>
+
+                    <div className="centered-buttons">
+                        <img className="img" src="/src/images/add_contact.png" alt="conversations"></img>
```

Comment:
I've observed that there seems to be a problem with displaying images in the app (At least on my end). 

To fix it, try the following solution:

- Go to the "public" directory and create an " images " folder in there.

- Place all visual assets in this newly created "images" folder.

- Whenever you need to use images in your components or pages, use this pattern for their paths:  `src="images/image_name.format"`
(For instance:`<img className="img" src="images/add_contact.png" alt="conversations"></img>`)

Let me know if it helped :)